llama_index/callbacks/global_handlers.py

```diff
@@ -17,6 +18,8 @@ def create_global_handler(eval_mode: str, **eval_params: Any) -> BaseCallbackHan
     """Get global eval handler."""
     if eval_mode == "wandb":
         handler: BaseCallbackHandler = WandbCallbackHandler(**eval_params)
```

Comment:
can we expose the `handler` object to the user?

`WandbCallbackHandler` got two methods - `persist_index` (to upload the index to artifacts) and `load_storage_context` (to download the index from the artifact).

Reply:
you can access it via `llama_index.global_handler`, can make that more explicit